db/fixtures/ae_datastore/ManageIQ/System/Event/EmsEvent/RHEVM.class/user_suspend_vm_ok.yaml

```diff
@@ -0,0 +1,16 @@
+---
+object_type: instance
+version: 1.0
+object:
+  attributes:
+    display_name:
+    name: USER_SUSPEND_VM_OK
```

Comment:
@lfu Do you know the difference between this event and `USER_SUSPEND_VM_FINISH_SUCCESS` which also calls the `vm_suspend` policy event for RHEVM?  Is it possible they could both be called and trigger the policy event twice for the same action?

Note: The `USER_SUSPEND_VM_FINISH_SUCCESS` event does not contain a call to `change_event_target_state`.

cc @oourfali